examples/dynamic_props/main.rs

```diff
@@ -26,16 +27,18 @@ fn resolve_project_points(facade: ResolverFacade<(), ()>) -> ExecutionResult {
     facade.resolve_scalar(points)
 }
 
-#[tokio::main]
```

Comment:
Curious - why the switch?

Reply:
This was the cause of #59.  When the engine, through the executor, calls the dynamic property resolver, the `ResolverFacade` creates a tokio runtime in order to interact with the Neo4J Bolt client. Because this example calls the engine executor inside a runtime context, the engine's attempt to create one causes a panic.

Once juniper releases an async version, such that the whole of the engine execute function can be wrapped in async, this should be fine.  For now, I took out the tokio execution of the main function and narrowed it down to a `block_on` call for the Neo4J client initialization.